### ISSUE TITLE
`yarn install` fails with 401 Unauthorized error 

### ISSUE THREAD

**Issue by micahstubbs** [uber-common/vis-academy#32]
`"https://unpm.uberinternal.com/ocular/-/ocular-0.2.16.tgz: Request failed \"401 Unauthorized\"".`

![screen shot 2017-10-13 at 4 02 55 pm](https://user-images.githubusercontent.com/2119400/31569478-0c76bc7e-b030-11e7-90e1-57c75bfeb2d2.png)

**Comment by micahstubbs** [comment]
in contrast `npm install` seemed to succeed

**Comment by akre54** [comment]
Can you blow away your yarn lockfile and retry? Looks like it's pointing to our internal npm repo.

**Comment by micahstubbs** [comment]
that did it 🎉 

![screen shot 2017-10-13 at 5 28 09 pm](https://user-images.githubusercontent.com/2119400/31570728-e7514336-b03b-11e7-9a5b-b92d7b91d516.png)

**Comment by akre54** [comment]
I think we should leave this open. It's a known issue for those of us who work for companies that use an internal npm registry on our work machines. 

It looks like it's related to yarnpkg/yarn#606, in which case it's fixable.

**Comment by vndprajapat8** [comment]
got the same issue but it got fixed for me by loging out of npm
**npm logout**

**Comment by Casey2085** [comment]
Had encountered similar issue while trying to install yarn globally. For working with the internal npm repo, had configured the npm.rc file. Removed the file from the users folder temporarily and was able to install yarn.

**Comment by jmurretxactly** [comment]
Ran into this going through your tutorial as well as yarn.lock has uberinternal registry.  It's easily fixable through deleting yarn.lock file.  It would be worth noting in the vis academy Getting Started as everyone outside of Uber will run into this.
Guys thanks for these tutorials!  You are doing awesome work here.

**Comment by amandakoster** [comment]
Had that issue, though deleted my yarn-lock file > yarn > yarn start. Solved it.

**Comment by localjo** [comment]
I also ran into this problem while going through [the setup for the kepler.gl tutorial](http://vis.academy/#/kepler.gl/setup). I tried deleting the `yarn.lock` and `package-lock.json` in the root of the repo and it *didn't* fix the problem. Then I realized there is also a `yarn.lock` in the tutorial subdirectory. Deleting that one fixed the issue. Mentioning it here in case anyone else is in the same boat as me and having trouble getting the above solution to work.
